static/js/src/sf_campaign-search.js

```diff
@@ -43,10 +43,11 @@ function setUpCampaignSearchField() {
           );
           if (response.ok) {
             const data = await response.json();
-            updateSearchResults(data);
+            updateSearchResults(data.campaigns);
           }
         } catch (error) {
           console.error("Error fetching campaign data:", error);
+          alert("Failed to fetch campaign data");
```

Comment:
I don't believe we use this kind of alert anywhere else on the site (or other sites), why did you choose to add it here?

Reply:
Idea was to let users know that for whatever reasons we were not able to get the campaign data so that they just not keep waiting for results to load up if there is an error.
Ideally automatically vanishing notifications would have provided a better UX but then they would have increased the scope of task.
Hence the alert.